### ISSUE TITLE
Improve PDF rendering quality

### ISSUE THREAD

**Issue by AkhilGopan** [wojtekmaj/react-pdf#444]
**Before you start - checklist**

 - [x] I have read documentation in README
 - [x] I have checked [sample](https://github.com/wojtekmaj/react-pdf/tree/master/sample) and [test](https://github.com/wojtekmaj/react-pdf/tree/master/test) suites to see real life basic implementation
 - [x] I have checked if this question is not already asked

**What are you trying to achieve? Please describe.**

Im trying to render some PDF with images and texts

Describe what are you trying to achieve Ex. I'd like to display multiple pages of my PDF.

I'm trying to render some PDF with images and texts with a different number of pages. When previewed in react-pdf, it renders in the viewer. But it loses the original quality on rendering. I would like to know if in any manner I can retain the original quality/improve the quality in viewing the pdf

**Describe solutions you've tried**

1. I have tried all sample codes related to this project and even tried your sample.pdf.
2. I have applied CSS to make it better, including Transform, Zoom options

**Additional information**

Screenshot attached
![image](https://user-images.githubusercontent.com/7835371/63720021-e032eb00-c86b-11e9-863b-2f79345ac921.png)

Above is when rendered in the viewer

![image](https://user-images.githubusercontent.com/7835371/63720291-7a932e80-c86c-11e9-9aa9-2c4288e4a443.png)

Above is when the pdf is originally viewed in a browser.

**Environment**

 - Browser (if applicable) Chrome 76
 - React-PDF version - Latest
 - React version - 16

**Comment by wojtekmaj** [comment]
It looks like you're using high DPI device and for some reason your PDF is still rendering with a "standard" DPI. That's quite odd - sample and test setups don't have this problem. Would you mind sharing code on CodeSandbox so I could have a look at your setup and potentially fix it?

**Comment by AkhilGopan** [comment]
> It looks like you're using high DPI device and for some reason your PDF is still rendering with a "standard" DPI. That's quite odd - sample and test setups don't have this problem. Would you mind sharing code on CodeSandbox so I could have a look at your setup and potentially fix it?

**Comment by AkhilGopan** [comment]
I have accidentally closed without making a comment. Thanks for the quick reply mate. I was putting css ZOOM: 1 for some other purposes causing this. Now after I made changes like,
`	this.setState({width: this.pdfWrapper.getBoundingClientRect().width}) `
I was able to render it much better.

Now, I have some related problems, which is related to my use cases on different screen resolutions (which is not related to this original issue.)

Thanks Again. This is really good.
